src/core/brand/roundel-filled.tsx

```diff
@@ -9,16 +9,10 @@ export const SvgRoundelFilled = ({
 }) => {
 	return (
 		<svg viewBox="0 0 42 42" xmlns="http://www.w3.org/2000/svg">
+			<circle cx="21" cy="21" r="21" fill={background} />
 			<path
-				fillRule="evenodd"
-				clipRule="evenodd"
-				d="M21.0001 0C9.4022 0 0 9.40185 0 21C0 32.598 9.4022 42 21.0001 42C32.5981 42 42 32.598 42 21C42 9.40185 32.5981 0 21.0001 0Z"
-				fill={background}
-			/>
-			<path
-				fillRule="evenodd"
-				clipRule="evenodd"
```

Comment:
Lazy Q: fillRule and clipRule get included by Figma when exporting SVGs. Are they doing anything useful, or can we just delete them every time?

Reply:
They can be useful in complex cut-out shapes.